src/main/java/seedu/savenus/logic/commands/SaveCommand.java

```diff
@@ -0,0 +1,44 @@
+package seedu.savenus.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.savenus.logic.commands.exceptions.CommandException;
+import seedu.savenus.model.Model;
+import seedu.savenus.model.savings.Savings;
+
+/**
+ * Saves an amount of money into the user's savings account.
+ */
+public class SaveCommand extends Command {
+
+    public static final String COMMAND_WORD = "save";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Set User's savings\n"
+            + "Parameters: SAVING_AMT\n"
+            + "Restriction: " + Savings.MESSAGE_CONSTRAINTS + "\n"
+            + "Example: " + COMMAND_WORD + " 100";
+
+    public static final String MESSAGE_SAVINGS_SUCCESS = "Current Savings: TODOOOOOOOOO";
+
+    private final Savings savingsAmount;
+
+
+
+    public SaveCommand(String savings) {
+        this.savingsAmount = new Savings(savings);
+    }
+
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
+        requireNonNull(model);
+
+        // add to the savings account in the model.
+        model.addToSavings(this.savingsAmount);
+
+        // deduct from wallet in model
+        model.deductFromWallet(this.savingsAmount);
```

Comment:
Maybe can consider swapping this around, coz if there is an exception thrown from deducting the amount from wallet, that amount shouldn't be added to the savings account

Reply:
Good idea! Thanks for the thorough review @Raikonen  👍